packages/e2e-test-app/test/LegacyImageTest.test.ts

```diff
@@ -14,27 +14,35 @@ beforeAll(async () => {
 describe('LegacyImageTest', () => {
   /* Test case #1: view and image displayed with no border and cornerRadius */
   test('ImageWithoutBorderTest', async () => {
-    const dump = await dumpVisualTree('image-container');
+    const dump = await dumpVisualTree('image-container', {
+      additionalProperties: ['Left', 'Top'],
+    });
     expect(dump).toMatchSnapshot();
   });
 
   /* Test case #2: Click button once, update view and image with round border*/
   test('ImageWithBorderTest', async () => {
     await toggleImageBorder();
-    const dump = await dumpVisualTree('image-container');
+    const dump = await dumpVisualTree('image-container', {
+      additionalProperties: ['Left', 'Top'],
+    });
     expect(dump).toMatchSnapshot();
   });
 
   /* Test case #3: Click button one more, remove border from view and image but tree sturcture is different from #1*/
   test('ImageWithoutBorderTestOneMoreClick', async () => {
     await toggleImageBorder();
-    const dump = await dumpVisualTree('image-container');
+    const dump = await dumpVisualTree('image-container', {
+      additionalProperties: ['Left', 'Top'],
+    });
     expect(dump).toMatchSnapshot();
   });
 
   test('ImageRTLTest', async () => {
     await toggleRTLMode();
-    const dump = await dumpVisualTree('image-container');
+    const dump = await dumpVisualTree('image-container', {
+      additionalProperties: ['Left', 'Top'],
```

Comment:
Instead of updating each of rhe callsites, could we do this in the library function? That way it is propagated everywhere.

Reply:
It's not possible to enumerate attached properties in XAML, and the lib is used by other projects as well, so it's necessary to pass a list of attached props you want it to extract via reflection.

Reply:
Can we pass this in dumpVisualTree?

Reply:
Passing this now via the entry point to the test app.